Formula/l/lima.rb

```diff
@@ -1,11 +1,18 @@
 class Lima < Formula
   desc "Linux virtual machines"
   homepage "https://lima-vm.io/"
-  url "https://github.com/lima-vm/lima/archive/refs/tags/v0.23.2.tar.gz"
-  sha256 "fc21295f78d717efc921f8f6d1ec22f64da82bfe685d0d2d505aee76c53da1ff"
   license "Apache-2.0"
   head "https://github.com/lima-vm/lima.git", branch: "master"
 
+  stable do
+    url "https://github.com/lima-vm/lima/archive/refs/tags/v0.23.2.tar.gz"
+    sha256 "fc21295f78d717efc921f8f6d1ec22f64da82bfe685d0d2d505aee76c53da1ff"
```

Comment:
(CI doesn't allow `url` and `sha256` to be present outside the `stable` block when the `stable` block exists)